<launch>

<arg name="resolution" default="0.1"/> <!-- 栅格边长 -->

<arg name="map_size_x" default="80.0"/>
<arg name="map_size_y" default="80.0"/>
<arg name="map_size_z" default=" 3.0"/> <!-- 地图尺寸 -->

<arg name="start_x" default=" 0.0"/>
<arg name="start_y" default=" 0.0"/> <!-- 起点 -->
<arg name="start_z" default=" 1.0"/>

<arg name="sight_radius" default=" 100.0"/> <!-- 视野半径 -->
<arg name="interval" default=" 0.5"/>    <!-- 轨迹段长度 -->

  <node pkg="grid_path_searcher" type="astar_node_3d" name="astar_node" output="screen" required = "true">
     <!-- FIXME 修改remap -->
      <remap from="~waypoints"       to="/move_base_simple/goal"/>
      <remap from="~map"             to="/sdf_map/occupancy"/> 
      <!-- <remap from="~pose"       to="/bs_aim"/> -->
      <!-- <remap from="~pose"       to="/mavros/local_position/pose_orb"/>   -->
      <remap from="~pose"            to="/fsm/planning_start"/> 
      <!-- <remap from="~real_pose"       to="/odom_visualization/pose"/>  -->
      <remap from="~real_pose"       to="/orb_odom"/> 


      <param name="map/resolution"   value="$(arg resolution)" />
      <param name="map/x_size"       value="$(arg map_size_x)"/>
      <param name="map/y_size"       value="$(arg map_size_y)"/>
      <param name="map/z_size"       value="$(arg map_size_z)"/>

      <param name="planning/start_x" value="$(arg start_x)"/>
      <param name="planning/start_y" value="$(arg start_y)"/>
      <param name="planning/start_z" value="$(arg start_z)"/>

      <param name="planning/sight_radius" value="$(arg sight_radius)"/>
      <param name="planning/interval"     value="$(arg interval)"/>

  </node>

</launch>
